packages/aws-cdk-lib/aws-cloudwatch/lib/alarm.ts

```diff
@@ -24,6 +24,20 @@ export interface AlarmProps extends CreateAlarmOptions {
   readonly metric: IMetric;
 }
 
+/**
+ * Properties for Anomaly Detection Alarms
+ */
+export interface AnomalyDetectionAlarmProps extends AlarmProps {
```

Comment:
The `AnomalyDetectionAlarmProps` extend the `AlarmProps`, but the `AnomalyDetectionAlarm` does not extend `Alarm`. 

Why is the `AnomalyDetectionAlarm` a separate construct? It looks like it's a strict superset of the `Alarm` we have already, which suggests that instead of making a new construct we could add a prop to `Alarm`. 

To make this decision we have to understand what the differences between an `AnomalyDetectionAlarm` and an `Alarm` are. Are there any properties unique to the `AnomalyDetectionAlarm` which conflict with the props of a regular `Alarm`? (Props 'A' and 'B' are said to conflict if I can (or should) specify 'A' or 'B', but not both).